src/app/routes/api/suggestionHandler.ts

```diff
@@ -49,9 +50,52 @@ export default function(requ : Request, resp : Response) : void {
 	};
 
 	log('Received comment from "%s"', email);
+	return new SuggestionModel(suggest).save();
```

Comment:
The database models must not be used directly, always go through the service layer:
```
import { suggestionService } from 'app/services';
...
suggestionService.save(...).then(...);
```